requirements-dev.txt

```diff
@@ -7,7 +7,8 @@ qiskit-sphinx-theme>=1.6
 sphinx-autodoc-typehints
 jupyter-sphinx
 pygments>=2.4
-reno>=3.2.0
+# TODO: Remove after reno 3.3.0 is released
```

Comment:
This should be:

```suggestion
# TODO: Remove after reno 3.4.0 is released
```

but it might be unnecessary because I just asked the maintainers again for a release and they proposed the patch to push it out here: https://review.opendev.org/c/openstack/releases/+/803227